### ISSUE TITLE
Not able to open saved .osp file, errors as invalid load key 

### ISSUE THREAD

**Issue by Lukn4Answers** [OpenShot/openshot-qt#2754]
**Describe the bug**
invalid load key

**Steps to reproduce the behavior:**
1. Opened Open Shot (after previous crashes, then PC updates, and again after deleting log files and downloading daily build).
2. Clicked "Open Project".
3. Selected the file from desktop (where it's been saved since creating project).
4. Error message appeared " Failed to load project file [filename.osp]: invalid load key, '\x00' ". (no double quotes or end period)

**Expected behavior**
Having not changed the path of the saved project, I expected to be able to open it and continue working on the file after troubleshooting OpenShot after crashes.

**System Details**
- OpenShot Version: 2.4.4-dev1
- libopenshot Version:  0.2.3
- Platform: Windows-10-10.0.17763
- Processor: Intel64 Family 6 Model 76 Stepping 3, GenuineIntel
- Machine: AMD64
- Python version: 3.6.4
- Qt5 version: 5.9.1
- PyQt5 version: 5.9
- Qt Detected Languages: ['en-US']
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build Installed: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.4.4-dev1-1555530828-aa85c1bb-9a1a6093-x86_64.exe

**Log Files**
- http://logs.openshot.org/43eqkekjfe4snanpp7vermpb9c0vi4ci/openshot-qt.zip
- http://logs.openshot.org/43eqkekjfe4snanpp7vermpb9c0vi4ci/libopenshot.zip

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by SuslikV** [comment]
Can you provide your project file?
Because the issue looks similar to this one: https://github.com/OpenShot/openshot-qt/issues/2733

**Comment by Lukn4Answers** [comment]
It is exactly the same issue, and it looked like it was addressed about 10 days ago, but since I'm experiencing also, maybe not as I thought. Deleting and starting over isn't a real option for me. Provide project file?

**Comment by Lukn4Answers** [comment]
![4 git file properties](https://user-images.githubusercontent.com/50082085/56898054-bbfa0580-6a55-11e9-9a78-63ec67f989eb.PNG)

**Comment by SuslikV** [comment]
Can you upload your project file (.osp) here (as zip archive)? If file contains only zeros, then just make new project, and make backups of your work manually.

**Comment by Lukn4Answers** [comment]
> Can you upload your project file (.osp) here (as zip archive)? If file contains only zeros, then just make new project, and make backups of your work manually.

@SuslikV @jonoomph I started a new project on 4/29, and then last night (4/30) it crashed. This morning was able to open project, but then OpenShot crashed again. When I tried to relaunch, got an error message that settings need to reset. After "OK", software appeared with welcome message. Opening my project failed, invalid load key. Ran the build .exe, tried to re-open my project, and again invalid load key. Starting a new project in this software is not an option.
